services/management/deps.go

```diff
@@ -22,6 +22,8 @@ import (
 
 	"gopkg.in/reform.v1"
 
+	"github.com/percona-platform/saas/pkg/check"
```

Comment:
Please move it up, to the same group as reform

Reply:
Imports still not sorted, there should be 3 groups of them

```
	"context"
	"time"

	"github.com/percona-platform/saas/pkg/check"
	"gopkg.in/reform.v1"

	"github.com/percona/pmm-managed/models"
```